source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc

```diff
@@ -0,0 +1,22 @@
+[[Configuring_timezones]]
+= Configuring the time zone
+
+{virt-product-fullname} stores time zone configurations for virtual machines in */etc/ovirt-engine/conf/00-timezone.properties*. This file contains default time zone values such as `Etc/GMT=Greenwich Standard Time`. It features mappings that are valid for windows and non-Windows time zones.
+
+[IMPORTANT]
+====
+Do not edit the actual *00-timezone.properties* file. Changes will be overwritten if you upgrade or restore the {engine-name}.
+
+Do not change values that come directly from the operating system or the {engine-name}.
+====
+
+To change the time zone configuration, create an override file in */etc/ovirt-engine/conf/*. The file name must begin with a value greater than `00`, so that the file appears after */etc/ovirt-engine/conf/00-timezone.properties*, and ends with the extension, *.properties*.
+
+For example, *10-timezone.properties* overrides the default file, *00-timezone.properties*. The last file in the file list has precedence over earlier files.
+
+Once you have created a time zone configuration file, new time zones can be added to that file. When adding new time zones, be sure the key is a valid General timezone from the link:https://en.wikipedia.org/wiki/Tz_database[tz database] and the value is a valid Windows timezone:
+
+General:: Time zones used for non-Windows OS types, must follow the standard tz format for example, `Etc/GMT` or `Asia/Jerusalem`.
+Windows:: Time zones specifically supported on link:https://docs.microsoft.com/en-us/previous-versions/windows/embedded/ms912391(v=winembedded.11)?redirectedfrom=MSDN[Windows] for example, `GMT Standard Time` or `Israel Standard Time`.
+
+For additional information about time zone formats, see the link:https://github.com/oVirt/ovirt-engine/blob/ovirt-engine-4.4.8.4/packaging/conf/timezones-defaults.properties#L12[time zone default properties file].
```

Comment:
I think the sentence should be remove - I don't remember seeing link to github on our documentation and it seems to me that the above covers it completely, no?

Reply:
> I think the last sentence should be remove - I don't remember seeing links to github in our documentation and it seems to me that the above covers it completely, no?

@ahadas Excellent point, I will remove the sentence. Thank you!